sources/subs/Recent.subs.php

```diff
@@ -268,4 +268,121 @@ function earliest_msg()
 	}
 
 	return $earliest_msg;
+}
+
+/**
+ * Callback-function for the cache for getLastTopics().
+ *
+ * @param mixed[] $latestTopicOptions
+ */
+function cache_getLastTopics($latestTopicOptions)
+{
+	return array(
+		'data' => getLastTopics($latestTopicOptions),
+		'expires' => time() + 60,
+		'post_retri_eval' => '
```

Comment:
> Notice: Serialization of 'Closure' is not allowed

The post_retri_eval is stored in the cache, that's the only reason I still have to ditch it.